src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import style from './style.scss';
+
+const cx = classNames.bind(style);
+
+class Progressbar extends React.Component {
```

Comment:
can you make this to stateless component? I think we don't need react component for this component since we are not using any `state`.

Reply:
You are right. React component seems a little bit heavy. I am thinking to export a function with those elements returned.
```js
function Progressbar() {
    ........
    return (
      <div>
        <div className={cx('progress-bar-container')}>
          <div style={{ width: `${progressBarWidth}%` }} className={this.getBarClass()}>
            {percentageLabel}
          </div>
        </div>
      </div>
    );
}

Reply:
please take a look at other components to see how stateless component is written. Also, there is good reference: https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc

Reply:
Got you. Thanks.